packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -478,10 +478,11 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
   // header scroll position
   useEffect(() => {
     if (getInteractionModality() === 'keyboard' && headerRef.current.contains(document.activeElement)) {
-      document.activeElement?.scrollIntoView?.({block: 'nearest', inline: 'nearest'});
+      scrollIntoView(headerRef.current, document.activeElement as HTMLElement);
+      scrollIntoViewport(document.activeElement, {containingElement: domRef.current});
```

Comment:
Needs these two called in succession so we first scroll the resizer into view w/ respect to the column header, and then we check if we need to scroll the entire table into the viewport if it is still isn't visible. If we omit the first scrollIntoView call, the table will always get centered in the viewport when resizing the last column to be larger.